py3-sphinxext-opengraph.yaml

```diff
@@ -0,0 +1,77 @@
+package:
+  name: py3-sphinxext-opengraph
+  version: 0.10.0
+  epoch: 0
+  description: sphinxcontrib-devhelp is a sphinx extension to generate unique OpenGraph metadata
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    provider-priority: "0"
+
+vars:
+  pypi-package: sphinxext-opengraph
+  import: sphinxext.opengraph
+
+data:
+  - name: py-versions
+    items:
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      3.13: '313'
+
+environment:
+  contents:
+    packages:
+      - py3-supported-build-base
+      - py3-supported-flit-core
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/sphinx-doc/sphinxext-opengraph
+      expected-commit: 935e43463e4fc985e56462a6c66f03a540d6c105
+      tag: v${{package.version}}
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: python${{range.key}} version of ${{vars.pypi-package}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
```

Comment:
please remove. this pattern is for transitioning from a previously non-multiversioned package. this package is net new.